tests/study_tests/test_study.py

```diff
@@ -799,9 +799,21 @@ def _objective(trial: Trial) -> float:
 def test_optimize_with_progbar_parallel(capsys: _pytest.capture.CaptureFixture) -> None:
 
     study = create_study()
-    with pytest.warns(UserWarning, match="Progress bar only supports serial execution"):
-        study.optimize(lambda _: 1.0, n_trials=10, show_progress_bar=True, n_jobs=-1)
+    study.optimize(lambda _: 1.0, n_trials=10, show_progress_bar=True, n_jobs=-1)
+    _, err = capsys.readouterr()
+
+    # search for progress bar elements in stderr
```

Comment:
A nit.
```suggestion
    # Search for progress bar elements in stderr.
```

Reply:
Ok, fixed both `test_optimize_with_progbar` and `test_optimize_with_progbar_parallel`.

I rebased and squashed these to one commit.

Reply:
Thank you!